schemas/add.yaml

```diff
@@ -0,0 +1,44 @@
+$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-sq/owl/blob/master/schema/add.yaml
```

Comment:
Good catch. (However, this directory and its contents are gone in the GraphQL version because GraphQL uses its own kind of schema which is written in Python and has no reference to github.)